src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderCancellationsDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments;
+using SwedbankPay.Sdk.PaymentInstruments.Card;
+using System;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    internal class PaymentOrderCancellationsDto
+    {
+        public string Id { get; set; }
+        public TransactionDto Cancellation { get; set; }
+
+        internal ICancellationsListResponse Map()

```

Comment:
It must be wrong that a singular cancellation class maps to a `List`?

```suggestion
        internal ICancellationResponse Map()
```

Reply:
`ICancellationResponse` is a different response with the properties `Uri Payment` and `ITransactionResponse Cancellation` while this creates a `IList<ITransaction> CancellationList`.

Reply:
Is this class tested? Can you make sure that the mapping actually works with real response data?

Reply:
There are tests in `src\Samples\Sample.AspNetCore.SystemTests\Test\PaymentTests\PaymentOrder\Standard\StandardPaymentOrderCancellationTests.cs`